### ISSUE TITLE
Second selector omitted when using & + &

### ISSUE THREAD

**Issue by mahtd** [sass/libsass#1441]
Looks like a regression in master

Input:

```
.foo {
    & + & {
        foo: bar;
    }
}
```

Ruby Sass output:

```
$ sass --version
Sass 3.4.16 (Selective Steve)
$ sass parent-sibling-parent.scss
.foo + .foo {
  foo: bar; }
```

libsass 3.2.5 output

```
$ sassc --version
sassc: 3.2.1
libsass: 3.2.5
sass2scss: 1.0.3
$ sassc parent-sibling-parent.scss
.foo + .foo {
  foo: bar; }
```

libsass master output

```
$ sassc --version
sassc: 3.2.5-14-g5c78
libsass: 3.2.5-172-g1ebed
sass2scss: 1.0.3
$ sassc parent-sibling-parent.scss
.foo + {
  foo: bar; }
```

**Comment by saper** [comment]
Probably broken with 3af051e6583a2d94cfdb6a51cb2f8dee3ebf0c22

**Comment by mahtd** [comment]
Yeah, I can confirm that

libsass @ `3af051e~1` (4767390)

```
$ sassc parent-sibling-parent.scss
.foo + .foo {
  foo: bar; }
```

libsass @ 3af051e

```
$ sassc parent-sibling-parent.scss
.foo + {
  foo: bar; }
```

**Comment by xzyfer** [comment]
Thanks for the investigation @saper and @mahtd

**Comment by xzyfer** [comment]
Spec added https://github.com/sass/sass-spec/pull/471

**Comment by xzyfer** [comment]
Fixed in #1465
